### ISSUE TITLE
Opening issue SHMF

### ISSUE THREAD

**Issue by chrbusch** [chrbusch/SHMF#1]
Moin! Sie sollten eine Email erhalten mit der Info, dass ich dies Ticket Ihnen "assigned" habe.
Dann bitte anrufen.

cbu

**Comment by SHMF-frankenberg** [comment]
War das jetzt schon das Ticket?

NF

**Comment by chrbusch** [comment]
Nein, dies ist ein "Übungs-Thread" für alle, die neu sind in github.
Jetzt, wo das klappt, können wir echte Tickets schreiben.

**Comment by SHMF-frankenberg** [comment]
Wunderbar. Lohnt es sich, GitHub auf den Desktop zu schmeißen, oder reicht der Browser?
